src/tests/ftest/control/dmg_system_leader_query.yaml

```
@@ -0,0 +1,8 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+timeout: 120
+server_config:
+    name: daos_server
```

```suggestion
  name: daos_server
```
Indentation is inconsistent.

---

Fixed